numerapi/base_api.py

```diff
@@ -893,3 +893,73 @@ def stake_increase(self, nmr, model_id: str = None) -> Dict:
              'status': ''}
         """
         return self.stake_change(nmr, 'increase', model_id)
+    
+    def set_payoutSelection(self, model_id: str = None, corrMultiplier: int = 0, 
+                            tcMultiplier: float = 0, takeProfit: bool = False, 
+                            selection: str = '', signals: bool = False) -> Dict:
+        """Change Payout Selection by Model.
+
+        Args:
+            model_id (str): Target model UUID (required for accounts with
+                multiple models)
+            corrMultiplier (int): 0 or 1
+            tcMultiplier (float): 0, 0.5, 1, 2, or 3
+            takeProfit (bool): determines whether payouts are returned to usr wallet 
+                or automatically staked to next round.
+            selection (str): tbh I have no idea what this does, empty str right now
+                doesn't seem to break anything
+            signals (bool): indicate whether model_id is for classic or signals model
+
+        Returns:
+           dict with confirmation that payout selection has been updated  
+
+        Example:
+            >>> api = NumerAPI(secret_key="..", public_id="..")
+            >>> model = api.get_models()['uuazed']
+            >>> api.set_payoutSelection(model, 1, 3)
+            {'data': {'v2ChangePayoutSelection': 'payout selection updated'}}
+
+        """
+        
+        #update these as the tournament allows different staking configurations
+        corrM_values = [0,1]
```

Comment:
I don't want to maintain these valid staking multipliers in numerapi. Either we get those from the API or we rely on numerai to validate them. To keep it simple, I would go for the latter. But both options are fine for me

Reply:
Makes total sense, by rely on numerai to validate do you mean simply leave it unconstrained and let numerai pass the error back if its an invalid value?

Reply:
Exactly.